### ISSUE TITLE
Access only to own clients, leads, SO, not following user profile

### ISSUE THREAD

**Issue by chrisz11** [Odoo-mobile/crm#39]
I did a few tests with the latest version of the app and so far main problem seems to be that apps gives an access only to it own contacts, sales order, etc. rather than following settings in the system like see own leads, etc. Especially for contacts not very suitable - can this can be configure or to be changed in the future?

**Comment by markusr** [comment]
I see the exact same problem. In my opinion it should be selectable whether to show all or only own contacts / sales orders / leads etc.

**Comment by chrisz11** [comment]
I guess sales orders and leads are covered by user profile on the server side but for example contacts may have 'salesman' field empty.

**Comment by FusionMK** [comment]
As this is the most important use for myself, using SO, Quotes on the move I think it is not really an enhancement need but it could be a bug. I use it on the Saas online version. It sometimes syncs some customers, even if salesman not set for that user, then another time it doesn't and not sure if all customers have been synced. ANother issue in the SO, when selecting qty it sometimes shows it in the order, and many times it just sets the order to blank.  Its suppose to work offline after being synced but in both cases, offline or online it does the same thing. Finally it constantly says there is an issue with login. asks to reset or remove account. There is  some connection security issue there, weather on the server side or app side not sure. Hope these issues can be checked soon. If it works the way I expect it to, this will be the best app for odoo out there, hands down.

**Comment by michaelkarrer81** [comment]
+1 it is not understandable for the average user the way it is

**Comment by falinwa** [comment]
yes, I agree, the access right should be the same on back end behavior. should not be different.It will confused the user. Still I think this is a good start. Good job!

**Comment by Priyanshi1010** [comment]
facing a problem,
user with see own leads option,can't see any leads 
only if see all leads option is enabled they can see all the leads created by administrator

give a solution if anyone knows
